egs2/tedx_spanish_corpus_openslr67/asr1/RESULTS.md

```diff
@@ -0,0 +1,29 @@
+<!-- Generated by scripts/utils/show_asr_result.sh -->
+# RESULTS
+## Environments
+- date: `Tue Jul 19 00:24:33 EDT 2022`
+- python version: `3.8.13 (default, Mar 28 2022, 11:38:47)  [GCC 7.5.0]`
+- espnet version: `espnet 202205`
+- pytorch version: `pytorch 1.11.0`
+- Git hash: `9afa4583518454ac867cb16b807115f1c1b3d169`
+  - Commit date: `Wed Jun 8 08:30:26 2022 +0000`
+
+## asr_train_asr_raw_bpe500_sp
```

Comment:
It would be great if you can also share the pre-trained model through huggingface (last stage in asr.sh)